BREAKING.md

```diff
@@ -74,6 +75,9 @@ The `maxMessageSize` property from `IConnectionDetails` and `IDocumentDeltaConne
 ### Remove `IntervalCollection.getView()` from sequence dds
 The `IntervalCollection.getView()` was removed.  If you were calling this API, you should instead refer to the `IntervalCollection` itself directly in places where you were using the view.
 
+### Added `ICodeDetailsLoader` and `IFluidModuleWithDetails` interface to `@fluidframework/container-definitions`
```

Comment:
Adding isn't a breaking change, but the impending removal from their old location is.  Prefer to lead with the breaking info so that customers can easily understand if they need to take action, e.g. "X & Y deprecated in package A, moving to package B".